src/platforms/react-native/sourcemaps/uploading/expo.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Expo
+description: "Upload source maps for native Expo releases and EAS Updates."
+---
+
+Sentry's React Native SDK works out of the box with Expo applications. To see readable stack traces in the product source maps need to be uploaded to Sentry. This guide explains how to upload source maps for Expo applications.
```

Comment:
```suggestion
Sentry's React Native SDK works out of the box with Expo applications. To see readable stack traces in the product, you must upload source maps to Sentry. This guide explains how to upload source maps for Expo applications.
```

Reply:
> Upload source maps for native Expo releases and EAS Updates

somewhat confusingly, the library is called `expo-updates` but the service is called "EAS Update"